sparse/tpls/KokkosSparse_spmv_mv_tpl_spec_decl.hpp

```diff
@@ -331,10 +331,208 @@ KOKKOSSPARSE_SPMV_MV_CUSPARSE(Kokkos::Experimental::half_t, int, int,
 #endif
 
 #undef KOKKOSSPARSE_SPMV_MV_CUSPARSE
-
 }  // namespace Impl
 }  // namespace KokkosSparse
 #endif  // defined(CUSPARSE_VERSION) && (10301 <= CUSPARSE_VERSION)
 #endif  // KOKKOSKERNELS_ENABLE_TPL_CUSPARSE
 
+// rocSPARSE
+#if defined(KOKKOSKERNELS_ENABLE_TPL_ROCSPARSE)
+#include "KokkosSparse_Utils_rocsparse.hpp"
+
+namespace KokkosSparse {
+namespace Impl {
+
+template <class Handle, class AMatrix, class XVector, class YVector>
+void spmv_mv_rocsparse(const Kokkos::HIP &exec, Handle *handle,
+                       const char mode[],
+                       typename YVector::non_const_value_type const &alpha,
```

Comment:
In general this looks a little funny, why not just: `typename YVector::const_value_type &alpha`?

Reply:
Good question, it was a copy-paste from another TPL. Fixed in all of them now